docs(docs-infra): Fix combobox dialog styles for dark theme

SkyZeroZx opened the pull request:
Fixes combobox dialog styles to ensure correct text color and contrast in dark theme. This improves readability and visual consistency when using the docs in dark mode.


 
## What is the current behavior?
Currently, it is not displaying correctly when we are in dark theme mode.
<img width="761" height="428" alt="image" src="https://github.com/user-attachments/assets/63225c38-b801-4244-b80f-55d0d04e57b7" />


## What is the new behavior?
<img width="823" height="405" alt="image" src="https://github.com/user-attachments/assets/88acfcfd-653a-44d4-90fa-ad125a92d388" />

JeanMeche commented:
When #66048 lands, I plan to isolate all our examples in a shadow DOM, where we don't have access to custom props defined in parents. 

Like I already suggested in #66050, we should probably use a combination of css custom props + a fallback value.

SkyZeroZx committed:
docs(docs-infra): Fix combobox dialog styles for dark theme

Fixes combobox dialog styles to ensure correct text color and contrast in dark theme.
This improves readability and visual consistency when using the docs in dark mode.

kirjs commented:
This PR was merged into the repository. The changes were merged into the following branches:

- main: 090484c85316a75397393112f6aae37763a18fba
- 21.0.x: c3adc5a707f378d9b3c755c9bfbccd89b0c68c20
